app/file.go

```diff
@@ -1126,7 +1126,7 @@ func (a *App) generateMiniPreview(fi *model.FileInfo) {
 		defer file.Close()
 		img, release, imgErr := prepareImage(a.srv.imgDecoder, file)
 		if imgErr != nil {
-			mlog.Debug("generateMiniPreview: prepareImage failed", mlog.Err(imgErr))
+			mlog.Debug("generateMiniPreview: prepareImage failed", mlog.Err(imgErr), mlog.String("FileInfo Id", fi.Id))
```

Comment:
```suggestion
			mlog.Debug("generateMiniPreview: prepareImage failed", mlog.Err(imgErr), mlog.String("fileinfo_id", fi.Id))
```

Just to keep some consistency with other logs.